testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -101,6 +108,9 @@
 COORDINATES_XYZ = resource_filename(__name__, 'data/coordinates/test.xyz')
 COORDINATES_XYZ_BZ2 = resource_filename(
     __name__, 'data/coordinates/test.xyz.bz2')
+COORDINATES_XTC = resource_filename(__name__, 'data/coordinates/test.xtc')
```

Comment:
This looks to me like I added the new test xtc file correctly to the test-suite. Also everything passes when I do a clean checkout of the repository on my laptop.